text/0000-target-tier-policy.md

```diff
@@ -0,0 +1,770 @@
+  Rust target except for the baseline expectations for the features or versions
+  of CPUs, operating systems, libraries, runtime environments, and similar,
+  then the proposed target must document to the satisfaction of the approving
+  teams why the specific difference in baseline expectations provides
+  sufficient value to justify a separate target.
+  - Note that in some cases, based on the usage of existing targets within the
+    Rust community, Rust developers or a target's maintainers may wish to
+    modify the baseline expectations of a target, or split an existing target
+    into multiple targets with different baseline expectations. A proposal to
+    do so will be treated similarly to the analogous promotion, demotion, or
+    removal of a target, according to this policy, with the same team approvals
+    required.
+    - For instance, if an OS version has become obsolete and unsupported, a
+      target for that OS may raise its baseline expectations for OS version
+      (treated as though removing a target corresponding to the older
+      versions), or a target for that OS may split out support for older OS
+      versions into a lower-tier target (treated as though demoting a target
+      corresponding to the older versions, and requiring justification for a
+      new target at a lower tier for the older OS versions).
+- Tier 2 targets must not leave any significant portions of `core` or the
+  standard library unimplemented or stubbed out, unless they cannot possibly be
+  supported on the target.
+  - The right approach to handling a missing feature from a target may depend
+    on whether the target seems likely to develop the feature in the future. In
+    some cases, a target may be co-developed along with Rust support, and Rust
+    may gain new features on the target as that target gains the capabilities
+    to support those features.
+  - As an exception, a target identical to an existing tier 1 target except for
+    lower baseline expectations for the OS, CPU, or similar, may propose to
+    qualify as tier 2 (but not higher) without support for `std` if the target
+    will primarily be used in `no_std` applications, to reduce the support
+    burden for the standard library. In this case, evaluation of the proposed
+    target's value will take this limitation into account.
+- The code generation backend for the target should not have deficiencies that
+  invalidate Rust safety properties, as evaluated by the Rust compiler team.
+  (This requirement does not apply to arbitrary security enhancements or
+  mitigations provided by code generation backends, only to those properties
+  needed to ensure safe Rust code cannot cause undefined behavior or other
+  unsoundness.) If this requirement does not hold, the target must clearly and
+  prominently document any such limitations as part of the target's entry in
+  the target tier list, and ideally also via a failing test in the testsuite.
```

Comment:
I think I mentioned this to josh in a separate conversation: As far as I know, the compiletest suite we use in the compiler does not have great support for registering failing tests that we *check* continue to fail. But maybe for the text here, an *ignored* failing test would suffice???

Reply:
If you duplicate it and mark it as check-fail with only-{target} that should work, I think.

Reply:
ah okay, my mistake then.